### PR TITLE
Fr localisation

### PR DESCRIPTION
**Pull request by NorXAengell**
French translation ;)

### CODE REVIEW

File: addons/unflipping/stringtable.xml [hunk 1]
```diff
@@ -9,6 +8,7 @@
         <Russian>Предел массы техники</Russian>
         <Spanish>Límite de masa de unidad</Spanish>
         <Chinesesimp>单位质量限制</Chinesesimp>
+        <French>Limite de masse individuelle </French>
```
Comment (veteran29) [inline addons/unflipping/stringtable.xml:11]:
```suggestion
        <French>Limite de masse individuelle</French>
```

File: addons/unflipping/stringtable.xml [hunk 2]
```diff
@@ -35,6 +35,7 @@
             设定一个单位可以翻正的物体的最大质量。
             默认值可使一个单位翻正大多数小型载具。
         </Chinesesimp>
+        <French>Définit le poid pouvant être retourné par une seule unité. La valeur par défaut devrait permettre à une seule unité de retourner la plupart des petits véhicules.  </French>
```
Comment (veteran29) [inline addons/unflipping/stringtable.xml]:
```suggestion
        <French>Définit le poid pouvant être retourné par une seule unité. La valeur par défaut devrait permettre à une seule unité de retourner la plupart des petits véhicules.</French>
```

File: addons/unflipping/stringtable.xml [hunk 3]
```diff
@@ -43,6 +44,7 @@
         <Russian>Необходимое кол-во людей</Russian>
         <Spanish>Límite de hombres</Spanish>
         <Chinesesimp>人数限制</Chinesesimp>
+        <French>Limite d'hommes </French>
```
Comment (veteran29) [inline addons/unflipping/stringtable.xml:47]:
```suggestion
        <French>Limite d'hommes</French>
```

File: addons/unflipping/stringtable.xml [hunk 4]
```diff
@@ -59,6 +62,7 @@
         <Russian>Время переворота</Russian>
         <Spanish>Tiempo para voltear</Spanish>
         <Chinesesimp>翻正时间</Chinesesimp>
+        <French>Temps de retournement </French>
```
Comment (veteran29) [inline addons/unflipping/stringtable.xml:65]:
```suggestion
        <French>Temps de retournement</French>
```

File: addons/unflipping/stringtable.xml [hunk 5]
```diff
@@ -75,6 +80,7 @@
         <Russian>Максимальный вес</Russian>
         <Spanish>Peso máximo para voltear</Spanish>
         <Chinesesimp>最大翻正重量</Chinesesimp>
+        <French>Poids maximum de retournement </French>
```
Comment (veteran29) [inline addons/unflipping/stringtable.xml:83]:
```suggestion
        <French>Poids maximum de retournement</French>
```

File: addons/unflipping/stringtable.xml [hunk 6]
```diff
@@ -113,6 +122,7 @@
         <Russian>Для переворота нужно: %1 чел.</Russian>
         <Spanish>Gente necesaria para voltear: %1</Spanish>
         <Chinesesimp>需要 %1 人才能翻正载具</Chinesesimp>
+        <French>Nombre de personnes pour retourner : %1</French>
```
Comment (veteran29) [inline addons/unflipping/stringtable.xml:125]: Is this space before ` :` wanted?
```suggestion
        <French>Nombre de personnes pour retourner: %1</French>
```

**Review by veteran29 — CHANGES_REQUESTED** [review]

**Comment by veteran29** [comment]
Thank you very much for submitting this translation!

I've found some minor issues, you can apply my suggestions in batch in "Files changed" tab

**Comment by NorXAengell** [comment]
Thx for your comments. 
In french from France there is a space before and after ponctuation like !?:
So between a word and ":" there's always a space before and after.
That why there a space at the end of translations.

However the correction is ok for the second. It's ok for you ?

**Comment by veteran29** [comment]
There's still some whitespace in settings names which seems unnecesary.

**Comment by NorXAengell** [comment]
Could you please say where ? ^^.
If for exemple here: <French>Limite de masse individuelle </French>
It's normal in french to have whitespace between word and ":" before and after it. 
The other way is a spelling error.

**Comment by veteran29** [comment]
I've just recognized that there is `:` after setting name in CBA. My bad.

**Review by veteran29 — APPROVED** [review]
Thanks again!

**Comment by NorXAengell** [comment]
No probleme thx to you :)

**Pull request merged**